syntax = "proto3";

service NotificationService {
    rpc SendNotification (NotificationRequest) returns (NotificationResponse);
}

message NotificationRequest {
    int32 user_id = 1;
    string message = 2;
}

message NotificationResponse {
    bool success = 1;
    string message = 2;
}